website/source/docs/cluster/bootstrapping.html.md

```diff
@@ -8,134 +8,189 @@ description: |-
 
 # Creating a cluster
```

Comment:
Before going into the "how-to" bootstrap a cluster, we should provide a brief overview of the components of the cluster and the network topology.